docs/guide/webpack.md

```diff
@@ -338,6 +340,43 @@ If a custom bundle imports packages that are also imported by the "main" bundle
 
 Apostrophe will take care of deploying the output files of custom bundles alongside those generated by the main bundle. Apostrophe will also generate the needed `script` and `link` tags automatically.
 
+### Rebundling resources
+
+In some cases, an npm package will provide front-end code that is either unbundled or bundled in a way that doesn't reflect the needs of the project. In these cases, the bundle configuration set by each module can be overridden through the `rebundleModules` option of the `@apostrophecms/asset` module.
+
+This option takes an object with module or module file names as properties and bundle names as values.
```

Comment:
This option takes an object with module names, optionally combined with a `:` and a bundle name, as properties